src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -232,6 +234,18 @@ namespace JSONHelpers
         auto result = json::from_file(save_file_path);
         if (result)
         {
+            if (!result->HasKey(L"app-zone-history"))
+            {
+                auto appZoneHistory = json::from_file(appZoneHistoryFilePath);
```

Comment:
`appZoneHistory` is parsed separately in `FancyZonesData::ParseAppZoneHistory`. I think it would be more readable if `appzoneHistory` will be read separately as well in method `json::JsonObject GetPersistAppZoneHistoryJSON()`

Reply:
It doesn't have to be in this PR, but I would also like to have the app zone history and the zone settings completely independent and avoid to merge the app zone history into the same json.
I understand that this was done to simplify backward compatibility, but at certain point we will drop the backward compatibility, so it's better to move in that direction already.

Reply:
Let's do it in another PR